src/manifolds/StiefelCanonicalMetric.jl

```diff
@@ -13,6 +13,17 @@ manifold, see[^EdelmanAriasSmith1998].
 """
 struct CanonicalMetric <: RiemannianMetric end
 
+"""
+    ApproximateLogarithmicMap <: ApproximateInverseRetraction
+
+An approximate implementation of the logarithmic map, which is an [`inverse_retract`](@ref)ion.
+See [`inverse_retract(::MetricManifold{ℝ,Stiefel{n,k,ℝ},CanonicalMetric}, ::Any, ::Any, ::ApproximateLogarithmicMap) where {n,k}`](@ref) for a use case.
+"""
+struct ApproximateLogarithmicMap{T} <: ApproximateInverseRetraction
+    max_iterations::T
```

Comment:
This could be just an integer.
```suggestion
    max_iterations::Int
```